background.html

```diff
@@ -32,12 +32,15 @@
       {{ /hideProgress }}
       <div class='message'>{{& message }}</div>
       <div>
-        {{ #exportButton }}
-          <button class='export grey'>{{ exportButton }}</button>
-        {{ /exportButton }}
-        {{ #debugLogButton }}
-          <button class='debug-log grey'>{{ debugLogButton }}</button>
-        {{ /debugLogButton }}
+        {{ #importButton }}
```

Comment:
These buttons might be a little opaque to the average non-exporting user (e.g. someone who is linking Signal-Desktop for the first time)

Maybe the second button could be prefaced with something like: "Already using Signal via the Chrome extension?"

In particular the confusion I'd want to avoid is an "existing" mobile app user thinking there is some way to export their old mobile data to Signal-Desktop.